packages/strapi-design-system/src/Link/Link.tsx

```diff
@@ -1,32 +1,25 @@
-import * as React from 'react';
+import React from 'react';
```

Comment:
What's the reason for this change? Why not do the same as in the CMS?

Reply:
Thanks for picking that up, I tried to catch them all through searching – obviously that didn't work very well - i'll redo 🤓

Reply:
oh I know why, because the V2 overrode the V1 so it looks like I changed it, but actually i just copied and pasted.